spring-boot-project/spring-boot-tools/spring-boot-buildpack-platform/src/main/java/org/springframework/boot/buildpack/platform/docker/TotalProgressListener.java

```diff
@@ -120,7 +120,7 @@ void finish() {
 		}
 
 		int getProgress() {
-			return withinPercentageBounds((this.progressByStatus.values().stream().mapToInt(Integer::valueOf).sum())
+			return withinPercentageBounds((this.progressByStatus.values().stream().mapToInt(Integer::intValue).sum())
```

Comment:
In that case `.reduce(0, Integer::sum)` can be used instead of `mapToInt(..).sum()`

Reply:
I think `.mapToInt(...).sum()` makes it easier to understand.